public/assets/homepage.js

```diff
@@ -203,6 +203,17 @@ $( document ).ready(function() {
         logWebpageActivity("Click_module=StartExploring_location=Index");
     });
 
+    //Triggered when the city or mapathon links are clicked
```

Comment:
Can you fix the comments you added to conform to the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide#general-guidelines)